src/main/resources/reference.conf

```diff
@@ -61,6 +61,11 @@ ergo {
 
     # Minimal fee amount of transactions mempool accepts
     minimalFeeAmount = 1000000
+
+    # A node is considering that the chain is synced if sees a block header with timestamp no more
+    # than headerChainTimeDiff blocks on average from future
```

Comment:
Please reformulate is simplier way, maybe with example.

Reply:
Improved description.